_targets.R

```diff
@@ -0,0 +1,32 @@
+library(targets)
source("code.R")
tar_option_set(packages = c("tidyverse", "sbtools", "whisker"))

```

Comment:
Hmm looks like something odd happened here and those `^M` characters got inserted where there should be  returns, so that the `library()` call, `source()` call and `tar_option_set()` call are each on new lines

Reply:
Ok ignore this, as it's fixed in your latest commit

Reply:
That's my bad. I did not push all the commits.